### DISCUSSION TITLE
articles/how-to-get-coding-project-idea

### DISCUSSION THREAD

**Discussion by giscus** [yehezkielgunawan/yehezgun-new#20] [Article Post]
# articles/how-to-get-coding-project-idea

Yehezkiel Gunawan's Article Post

https://yehezgun.com/articles/how-to-get-coding-project-idea

**Comment by marbonka** [comment]
On point with very practical standpoint. I think there are a lot of things that we can explore to build up our own portfolio. Either it is personal daily life, professional challenges or business use cases can be developed to showcase our ability to interpret and implement it into a UI/UX project app. I really love this writing and shout-out a salute to you!

**Reply by yehezkielgunawan** [reply]
Thanks you. Yeah, even it's just a small or simple tips, I believe it can help or inspire you and the readers. Don't give up, it took a while, but it will works if you keep it consistently :D
